modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -101,18 +101,26 @@ def phone_string
     end
 
     def area_code
+      return unless domestic_phone?
```

Comment:
These are still valid parts of international number. We shouldn't be gating them to only domestic numbers.

Reply:
K! Going to rework this into form_data so the area code doesn't print an extra area code on the form.